charts/redpanda/templates/console/configmap-and-deployment.yaml

```diff
@@ -1,4 +1,4 @@
-{{/*
```

Comment:
was this removed accident? If not then we need to do the same on the other end of the boilerplate

Reply:
Sorry, that was accidental. I have fixed the commit now.